tempodb/backend/raw.go

```diff
@@ -235,16 +235,16 @@ func KeyPathWithPrefix(keypath KeyPath, prefix string) KeyPath {
 }
 
 // MetaFileName returns the object name for the block meta given a block id and tenantid
-func MetaFileName(blockID uuid.UUID, tenantID string) string {
-	return path.Join(RootPath(blockID, tenantID), MetaName)
+func MetaFileName(blockID uuid.UUID, tenantID string, prefix string) string {
```

Comment:
what happens if prefix is empty. does this do what is expected? maybe add some quick tests?

Reply:
tests are at `tempodb/backend/raw_test.go` check the commit [add tests for modifed metaFile/Path methods](https://github.com/grafana/tempo/pull/2591/commits/3051385bd0f91c18ecfd17fac65dd94bae83b36a)